pkgs/by-name/ai/aider/package.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, python3
+, fetchFromGitHub
+, portaudio
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "aider";
+  version = "0.41.0";
+
+  src = fetchFromGitHub {
+    owner = "paul-gauthier";
+    repo = "aider";
+    rev = "v${version}";
+    hash = "sha256-OD2xGQvYFyiAnkRtb0VczGLvgWmW45n0HzdHoLLMdHY=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    backoff
+    beautifulsoup4
+    configargparse
+    diff-match-patch
+    diskcache
+    flake8
+    gitpython
+    google-generativeai
+    grep-ast
+    grep-ast
```

Comment:
```suggestion
    grep-ast
    ijson
```

Reply:
Is there a reason to add `ijson` here? I don't see it as a dependency upstream.

Reply:
It was required (from what I saw) to build on 0.41.0. Maybe not needed anymore.